### DISCUSSION TITLE
Server crashes when loading a project with AG insert

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#761] [Help]
Hi,

So I try to load several projects running with beta 4 and beta 5. The remote mac server crashes when loading AG plugins.

Those projects were not crashing the server under beta 4 and beta 5.

I sent you some log and traces by email.

**Comment by apohl79** [comment]
Hm looks like the server dies sometimes when booting up during the scan phase. Could you wipe the servers plugin cache file (~/.audiogridder/audiogridderserver.cache) and see if it can rescan your plugins?
